SigAverager/SigAverager/cwg.py

```diff
@@ -0,0 +1,75 @@
+"""Carrier Wave (CW) Generator for FEngine."""
+import numpy as np
+import scipy.stats
+
+
+def generate_carrier_wave(
+    cw_scale: float,
+    freq: float,
+    sampling_frequency: int,
+    num_samples: int,
+    noise_scale: float,
+    complex: bool,  # noqa: E501
+) -> np.ndarray:
+    """Generate a carrier wave vector.
+
+    Parameters
+    ----------
+    cw_scale: float
+        factor to scale generated noise.
+    freq: float
+        Frequency of CW to be generated.
+    sampling_frequency: int
+        Sample rate for generated CW. This is expressed in Hz. E.g. 1712e6.
+    num_samples: int
+        Number of samples for generated CW.
+    noise_scale: float
+        Factor to scale generated noise.
+    complex: bool
+        Specify if realor complex carrier wave is required.
+
+    Returns
+    -------
+    np.ndarray of type float
+        Output array of complex-valued samples for generated CW.
```

Comment:
```suggestion
        Complex-valued samples for generated CW.
```
You know it's an output, because it's the return value. You also know it's an array because the type-hint is there.

Reply:
Noted.